src/sidebar/app/actions.js

```diff
@@ -101,7 +101,8 @@ export function createNote(content = '', origin) {
     id,
     content,
     origin,
-    lastModified: new Date()
+    lastModified: new Date(),
```

Comment:
kinto.js is supposed to add it as soon and only as soon as the record is created remotely.

Reply:
This is confusing I agree, but we keep inside our blob an other lastModified date for the UI which is only updated by our redux reducer. Sync event was sometimes changing dates and that was confusing for the user. Also non-synced object had no dates ... so that why. We could have done differently ? 🤔

Reply:
Oh yeah you are right `lastModified` and `last_modified`

Reply:
Quick vote @Natim , should I rename it lastEdited to avoid any confusion ?

Reply:
lastModifiedDate ?